app/controllers/users/auth_app_controller.rb

```diff
@@ -0,0 +1,66 @@
+module Users
+  class AuthAppController < ApplicationController
+    include ReauthenticationRequiredConcern
+
+    before_action :confirm_two_factor_authenticated
+    before_action :confirm_recently_authenticated_2fa
+    before_action :set_form
+    before_action :validate_configuration_exists
+
+    def edit; end
+
+    def update
+      binding.pry
```

Comment:
```suggestion
```

Reply:
🤦🏾‍♀️ oops! Thanks for pointing this out.